entrypoint.sh

```diff
@@ -1,5 +1,11 @@
 #!/usr/bin/env bash
 
+# check codecov token exist.
+if [[ ! -n $PLUGIN_TOKEN ]] && [[ ! -n $CODECOV_TOKEN ]]; then
```

Comment:
Please change this to `if [[ -z $PLUGIN_TOKEN ]] && [[ -z $CODECOV_TOKEN ]]; then`